assets/src/design-system/components/mediaInput/index.js

```diff
@@ -156,7 +157,8 @@ export const MediaInput = forwardRef(function Media(
   {
     className,
     onBlur,
-    onChange,
+    // eslint-disable-next-line no-unused-vars
+    onChange = noop,
```

Comment:
You can completely remove the `onChange` prop here. It's never used.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/6900/commits/28c39ba7c4a1d05799b1b6b2070815e1c992cf00